llvm/test/CodeGen/SystemZ/builtin-setjmp-spills-double-02.ll

```diff
@@ -0,0 +1,145 @@
+; Simulate register pressure  around setjmp call for double precision arguments.
```

Comment:
Register pressure may indeed be a good reason to have a longer test.  Still, the CHECK statements should be auto-generated.  Also, even here we should be removing unnecessary attributes and annotations that are not essential to the purpose of the test.

Reply:
There doesn't seem to be a -01 test with this name, please rename to remove those gaps.